src/ASM/GlbL2projector.h

```diff
@@ -147,15 +147,22 @@ class GlbL2 : public Integrand
                            LocalIntegral& elmInt);
   //! \brief Initializes current element for numerical integration (mixed).
   //! \param[in] MNPC1 Matrix of nodal point correspondance for current element
+  //! \param[in] fe Nodal and integration point data for current element
   //! \param[in] elem_sizes Size of each basis on the element
   //! \param[in] basis_sizes Size of each basis on the patch
   //! \param elmInt Local integral for element
   virtual bool initElement(const IntVec& MNPC1,
+                           const MxFiniteElement& fe,
                            const uIntVec& elem_sizes,
                            const uIntVec& basis_sizes,
                            LocalIntegral& elmInt);
 
   //! \brief Dummy implementation.
+  virtual bool initElement(const IntVec& MNPC1,
+                           const uIntVec& elem_sizes,
+                           const uIntVec& basis_sizes,
+                           LocalIntegral& elmInt) { return false; }
```

Comment:
Is this dummy strictly necessary if you changed all the `initElement` calls in the ASM*mx* methods?

Reply:
yes. GlbL2projector inherits Integrand only, where this is pure virtual.

Reply:
OK, the alternative would be to remove the Integrand member also, but then all other downstreams using mixed integrands would be affected. Guess you want to avoid that..